### PR TITLE
raftstore: Report is apply busy to PD

### PR DESCRIPTION
**Pull request by Connor1996**
### What is changed and how it works?

Part of #11409  

What's Changed:
- Report is apply busy(apply wait duration > 100ms) to PD

### Check List 

- Manual test (add detailed scripts or steps below)

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore/src/store/fsm/apply.rs [hunk 1]
```diff
@@ -3627,9 +3629,11 @@ where
 
             match msg {
                 Msg::Apply { start, mut apply } => {
-                    apply_ctx
-                        .apply_wait
-                        .observe(start.saturating_elapsed_secs());
+                    let elapsed = start.saturating_elapsed_secs();
+                    if elapsed > 0.1 {
```
Comment (BusyJay) [inline components/raftstore/src/store/fsm/apply.rs:3633]: Raftstore consider 5 secs as busy.

File: components/raftstore/src/store/fsm/store.rs [hunk 2]
```diff
@@ -656,7 +657,9 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport> RaftPoller<EK, ER, T> {
     fn flush_events(&mut self) {
         self.flush_ticks();
         self.poll_ctx.raft_metrics.flush();
-        self.poll_ctx.store_stat.flush();
+        if self.poll_ctx.store_stat.flush() {
+            store_heartbeat_pd(&mut self.poll_ctx);
```
Comment (BusyJay) [inline components/raftstore/src/store/fsm/store.rs:661]: How about checking `is_busy` flags directly?

**Comment by Connor1996** [comment]
Implemented in #16239

**Pull request closed without merging**